translations/ja.yml

```diff
@@ -352,19 +354,19 @@ src:
           '3590336714': エクスポート
       Settings:
         Account:
-          '3198092477': アカウント
+          '3198092477': 'アカウント'
```

Comment:
Not sure the ' ' will show up in app, but just to be sure and to keep consistent let's remove them ;)